### PR TITLE
Rewrite of Cold Start positioning per Alok and various fixes

### PR DESCRIPTION
**Pull request by NWeber-SL**

### CODE REVIEW

File: introduction/requirements.md [hunk 1]
```diff
@@ -114,10 +114,10 @@ System | Windows
...
 
-ShiftLeft supports the following browsers for accessing, viewing and interacting with the ShiftLeft Dashboard:
+ShiftLeft supports the following browsers for accessing, viewing and interacting with the [ShiftLeft Dashboard](products.md):
 
-- Google Chrome (tested with v61, v62)
-- Mozilla Firefox (tested with v57, v58b)
-- macOS Safari (tested with v11)
+* Google Chrome (tested with ShiftLeft Protect v61, v62)
+* Mozilla Firefox (tested with ShiftLeft Protect v57, v58b)
+* MacOS Safari (tested with ShiftLeft Protect v11)
```
Comment (vlad-alexandru-ionescu) [inline introduction/requirements.md]: `tested with ShiftLeft Protect v...` doesn't make sense. Those are the versions of the browsers that our UI works with, not of Protect.

File: introduction/requirements.md [hunk 2]
```diff
@@ -114,10 +114,10 @@ System | Windows
...
 
-Microsoft Edge* is currently not officially supported but has been verified with v41.
+Microsoft Edge* is currently not officially supported but has been verified with ShiftLeft Protect v41.
```
Comment (vlad-alexandru-ionescu) [inline introduction/requirements.md]: Same here

File: introduction/requirements.md (hunk 1 above)
Comment (NWeber-SL) [reply introduction/requirements.md]: So should I just removed the "(tested with ...)"? Or does the version number reflect the version of the browser? This was part of the original content, so I don't know why or what this means.

File: introduction/requirements.md (hunk 2 above)
Comment (NWeber-SL) [reply introduction/requirements.md]: So should I just removed the "(tested with ...)"? Or does the version number reflect the version of the browser? This was part of the original content, so I don't know why or what this means.

**Pull request merged**